manticore/ethereum/manticore.py

```diff
@@ -328,7 +328,7 @@ def _compile(source_code, contract_name, libraries=None, solc_bin=None, solc_rem
         name = name.split(':')[1]
 
         if contract['bin'] == '':
-            raise EthereumError('Solidity failed to compile your contract.')
+            raise EthereumError('Solidity failed to generate bytecode for your contract. Check if all the abstract functions are implemented')
```

Comment:
> Check if all the abstract functions are implemented
is this always the case? if not, maybe this:
```suggestion
            raise EthereumError('Solidity failed to generate bytecode for your contract. It might be that you forgot to implement all of the abstract functions?')
```
Also, can't we print out warnings/errors from solc here? (maybe the `warnings` variable have them?)